src/korma_encrypted/core.clj

```diff
@@ -28,13 +28,25 @@
     (korma.db/with-db db
       (generate-and-save-data-encryption-key key-service))))
 
+(defn rotate-key-encryption-keys
+  ([old-service new-service]
+    (doseq [encryption-key (korma/select data-encryption-keys)]
```

Comment:
This should be wrapped in a transaction. It would be bad to end up in a state where some of the keys are encrypted under a different service. Alternatively, you could somehow tag each key to know what service it comes from.